### PR TITLE
Added Border color  style property to Container widgets

### LINKED ISSUE #4164: Add border color style property to Container widgets
## Proposed feature
Option to change the border color of the container widget with the `Border color` property in the style tab.

### Description
Currently, the container widget only supports **Background color** , **Border radius**, **Visibility** and **Disable** as style definition.

Screenshot 2022-10-01 at 9 56 46 AM

reference: https://github.com/ToolJet/ToolJet/pull/1721

### PR DESCRIPTION
**Pull request by prateek565**
closes #4164

### CODE REVIEW

**Comment by prateek565** [comment]
closes #4164

File: frontend/src/Editor/Components/Container.jsx [hunk 1]
```diff
@@ -12,14 +12,15 @@ export const Container = function Container({
...
-  const { visibility, disabledState, borderRadius } = styles;
+  const { visibility, disabledState, borderRadius,borderColor } = styles;
   const backgroundColor =
     ['#fff', '#ffffffff'].includes(styles.backgroundColor) && darkMode ? '#232E3C' : styles.backgroundColor;
   const computedStyles = {
     backgroundColor,
     height,
     display: visibility ? 'flex' : 'none',
     borderRadius: borderRadius ? parseFloat(borderRadius) : 0,
+    borderColor,
```
Comment (arpitnath) [inline frontend/src/Editor/Components/Container.jsx:23]:
```suggestion
    border: "1px solid",
    borderColor,
```

Comment (arpitnath) [reply frontend/src/Editor/Components/Container.jsx:23]: OR
```suggestion
border: `1px solid ${borderColor}`,
```

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 2]
```diff
@@ -2053,6 +2060,7 @@ export const widgets = [
         borderRadius: { value: '0' },
         visibility: { value: '{{true}}' },
         disabledState: { value: '{{false}}' },
+        borderColor: { value: '' },
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:2063]:
```suggestion
        borderColor: { value: '#fff' },
```

**Review by arpitnath — CHANGES_REQUESTED** [review]
Screenshot 2022-10-05 at 3 24 46 AM
Also, can we have the border color property next to **border radius**?

**Comment by prateek565** [comment]
Okayy i will get it done by today . Thanks for feedback 😊.

**Comment by prateek565** [comment]
hi @arpitnath  i have done all the changes as recomended.

**Review by arpitnath — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @prateek565  
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**